docs/pages/develop/user-interface/next-steps.mdx

```diff
@@ -1,12 +1,20 @@
 ---
 title: Next steps
-description: A list of useful resources to learn more about UI.
+description: A list of useful resources to learn more about implementing navigation and UI in your app.
 hideTOC: true
 ---
 
 import { BoxLink } from '~/ui/components/BoxLink';
+import { RouterLogo } from '@expo/styleguide';
 import { Brush01Icon, Settings02Icon, TableIcon } from '@expo/styleguide-icons';
 
+<BoxLink
+  title="Expo Router"
+  Icon={RouterLogo}
+  description="Expo Router is an open-source routing library for Universal React Native applications built with Expo."
```

Comment:
I like getting the "file system" part in there somewhere, see the description in https://www.youtube.com/watch?v=608r8etX_cg&t=678s
```suggestion
  description="Expo Router is an open-source file system-based routing library for Universal React Native applications built with Expo."
```